test/test_BE.js

```diff
@@ -0,0 +1,48 @@
+const tape = require ('tape');
+const supertest = require ('supertest');
+const app = require ('./../src/app.js');
+//test the home route
+tape('test for home route ', (t) => {
+    supertest(app)
+      .get('/')
+      .expect(200)
+      .end((err, res) => {
+        if (err) {
+          t.error(err);
+        }
+        t.ok(res.text.includes('body'));
+        t.equal(res.text.substr(0, 15), '<!DOCTYPE html>', 'the respone  should be html file');
+        t.equal(res.header['content-type'], 'text/html; charset=utf-8', 'the content-type should equal html');
+        t.end();
+      });
+  });
+  //test the sign-in route
```

Comment:
Is the sign up/ in part of the home issue?
remove these tests  and upload it with its own issue